.github/workflows/ci-testing-legacy.yml

```diff
@@ -0,0 +1,30 @@
+name: CI-Code-Style
+
+on:
+  push:
+    branches:
+      - "master"
+  pull_request:
+
```

Comment:
I think we want to run the CI on PRs. There are generally not that many

Reply:
I can't follow you ? This is what the workflow definition does, it's called on any pull request and it operates on master after merge. You can see that it does run on pull requests, because it ran on this pull request :)